pkg/experiment/metrics/rules.go

```diff
@@ -48,5 +87,101 @@ func recordingRulesFromTenant(tenant string) []*RecordingRule {
 			},
 			keepLabels: []string{},
 		},
+		{
+			profileType: "process_cpu:cpu:nanoseconds:cpu:nanoseconds",
+			metricName:  "pyroscope_exported_metrics_mimir_dev_10_ingester",
+			matchers: []*labels.Matcher{
+				{
+					Type:  labels.MatchEqual,
+					Name:  "service_name",
+					Value: "mimir-dev-10/ingester",
+				},
+			},
+			keepLabels: []string{"controller_revision_hash"},
+		},
+		{
+			profileType: "process_cpu:cpu:nanoseconds:cpu:nanoseconds",
+			metricName:  "pyroscope_exported_metrics_mimir_dev_10_ingester_with_span_names",
+			matchers: []*labels.Matcher{
+				{
+					Type:  labels.MatchEqual,
+					Name:  "service_name",
+					Value: "mimir-dev-10/ingester",
+				},
+			},
+			keepLabels: []string{"controller_revision_hash", "span_name"},
+		},
+	}
+}
+
+func SettingsClient() settingsv1connect.SettingsServiceClient {
+	// TODO: refactor this. Very rudimentary, only intended for experimental
+	httpClient := util.InstrumentedDefaultHTTPClient()
+	opts := connectapi.DefaultClientOptions()
+	opts = append(opts, connect.WithInterceptors(tenant.NewAuthInterceptor(true)))
+	return settingsv1connect.NewSettingsServiceClient(
+		httpClient,
+		"http://fire-tenant-settings-headless.fire-dev-001.svc.cluster.local:4100", // TODO: get it from config (set it in deployment tools) use "http://localhost:4040" for local
+		opts...,
+	)
+}
```

Comment:
Initialization should be done in `modules` and the `SettingsClient` component should to be injected as a dependency